app/src/views/animations/status-animations.js

```diff
@@ -47,10 +48,38 @@ export default class StatusAnimations {
       }
       direction *= -1;
     }
+    frames.push(() => [[lastPanel, 0, "error"]]);
 
     StatusAnimations.animateStrips(view, ['0', '1', '2'], frames, 50, completeCallback);
   }
 
+  // FIXME: This is a hack to support failure animation on one panel
```

Comment:
Do you see any way we can fix this immediately with minimal effort? If not, I'm thinking that we leave this as a fix for later when we begin incorporating more of the things from views into game logic.